static/js/util/util.js

```diff
@@ -378,7 +378,13 @@ export function formatPrice(price: ?string|number|Decimal): string {
   if (price === null || price === undefined) {
     return '';
   } else {
-    return `$${price}`;
+    let formattedPrice: Decimal = Decimal(price);
+    if (!formattedPrice.isInteger()) {
+      formattedPrice =  formattedPrice.toFixed(2, Decimal.ROUND_HALF_UP);
+    }
+
+    let dollars = `$${formattedPrice}`;
+    return dollars.replace(/\.00$/,'');
```

Comment:
Please don't use a regular expression in this function. Just use `formattedPrice.toFixed(2)` if `formattedPrice.isInteger()` else `formattedPrice.toFixed(0)`